packages/devp2p/src/dns/dns.ts

```diff
@@ -37,7 +36,7 @@ export class DNS {
   constructor(options: DNSOptions = {}) {
     this._DNSTreeCache = {}
 
-    if (isTruthy(options.dnsServerAddress)) {
+    if (options.dnsServerAddress !== undefined) {
```

Comment:
Since we're expecting this to be `string` type, would it make more sense to explicitly check for it being a string rather than just verifying it's not undefined?

Reply:
Yep, I think that's abetter